src/main/java/org/broadinstitute/consent/http/db/DACUserMapper.java

```diff
@@ -10,12 +10,13 @@
 public class DACUserMapper implements ResultSetMapper<DACUser> {
 
     public DACUser map(int index, ResultSet r, StatementContext ctx) throws SQLException {
-
-        return new DACUser(
-                r.getInt("dacUserId"),
-                r.getString("email"),
-                r.getString("displayName"),
-                r.getDate("createDate"),
-                r.getString("additional_email"));
+        DACUser user = new DACUser();
+        user.setDacUserId(r.getInt("dacUserId"));
+        user.setEmail(r.getString("email"));
+        user.setEmailPreference(r.getBoolean("email_preference"));
```

Comment:
Would be nice to consistently have camelCase or snake_case.. not sure if it's worth the migration here. I know in Postgres the convention is to go with snake_case for column/table names.

Reply:
Yes, you're seeing the reason why I'm going through a lot of refactoring. I have https://broadinstitute.atlassian.net/browse/DUOS-282 to deal with this more generically, but am addressing these problems in a piecemeal fashion. We're moving to snake case db fields as we add new ones and there will be more refactoring to move camel cased columns to snake case.